gallery_dl/extractor/imagefap.py

```diff
@@ -19,7 +19,7 @@ class ImagefapExtractor(Extractor):
     category = "imagefap"
     root = "https://www.imagefap.com"
     directory_fmt = ("{category}", "{gallery_id} {title}")
-    filename_fmt = "{category}_{gallery_id}_{filename}.{extension}"
+    filename_fmt = "{category}_{gallery_id}_{num:07}_{filename}.{extension}"
```

Comment:
Padding the number to 7 places is too much, I think. There might be a few galleries with 10k or even 100k images, but they are an exception. For more than 99% of galleries, padding to 3 or 4 places will be enough.

Reply:
Sounds good.